velox/common/base/ConfigUtil.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <re2/re2.h>
+
+#include "velox/common/base/ConfigUtil.h"
+
+namespace facebook::velox::config {
```

Comment:
I assume there is no functional change? Why not put this into velox/common/config/config.h?

Reply:
That is a completely different ConfigBase class that is only used by dwrf and has nothing to do with this. But it will be nice to do some followup cleanup for our configs.